openedx/openedx-learning
# Documents Tagging architecture decisions

This PR documents the decisions that arose from https://github.com/openedx/modular-learning/issues/60.

It also builds on https://github.com/openedx/openedx-learning/pull/54 and enforces [ADR 7. Tagging app structure](https://github.com/open-craft/openedx-learning/blob/jill/tagging-data-arch-adr/docs/decisions/0007-tagging-app.rst) by moving the tagging app to a separate package outside of `openedx_learning`, called `openedx_tagging`. The models in this app will be altered and aligned with the other ADRs as part of subsequent PRs.

**Testing instructions**

1. When running the automated tests, check that the `openedx_tagging` test runs.
1. Check the added ADRs for clarity and correctness.

**Supporting information**

Closes https://github.com/openedx/modular-learning/issues/60
